corehq/apps/app_manager/management/commands/fix_title_label.py

```diff
@@ -0,0 +1,31 @@
+from corehq.toggles import SYNC_SEARCH_CASE_CLAIM
+
+from django.core.management.base import BaseCommand
+
+from corehq.apps.app_manager.models import Application
+from corehq.apps.app_manager.management.commands.helpers import get_all_app_ids
+
+
+class Command(BaseCommand):
+
+    def handle(self, **options):
+        domains = sorted(SYNC_SEARCH_CASE_CLAIM.get_enabled_domains())
+        for domain in domains:
+            print(f"Looking at domain: {domain}")
+            app_ids = get_all_app_ids(domain)  # , include_builds=True)
+            for app_id in app_ids:
+                app = Application.get(app_id)
+                print(f"Checking Application Modules: {app_id}")
+                for module in app.modules:
+                    if hasattr(module, 'search_config'):
+                        print(f"Looking at Module: {module.name[app.default_language]}")
+                        try:
+                            title_label = module.search_config.title_label
+                            for lang, string in title_label.items():
+                                if string is None:
+                                    print(f"Fixing translation: {{{lang}: {string}}}")
+                                    module.search_config.title_label[lang] = ""
+                                    app.save()
```

Comment:
Can you move this outside of the `modules` loop?  As-is, this would save multiple times per app (once per language, per search configuration)

Reply:
When I tested locally I wasn't able to save the changes like this when I moved `save()` out of the `modules` loop. I guess an alternative could be this 
```python3
for module in app.modules:
  if hasattr(module, 'search_config'):
      print(f"Looking at Module: {module.name[app.default_language]}")
      title_label = module.search_config.title_label
      new_title_label = {}
      for lang, string in title_label.items():
          if string is None:
              print(f"Fixing translation: {{{lang}: {string}}}")
              new_title_label[lang] = ""
          else:
              new_title_label[lang] = string
      module.search_config.title_label = new_title_label
app.save()
```

Reply:
yep, that's about what I was imagining - looks like it should work